framework/wazuh/rbac/default/policies.yaml

```diff
@@ -143,6 +143,28 @@ default_policies:
           - list:path:*
         effect: allow
 
+  rootcheck_read:
+    description: Allow read rootcheck information
+    policies:
+      syscheck:
+        actions:
+          - rootcheck:read
+        resources:
+          - agent:id:*
+        effect: allow
+
+  rootcheck_all:
+    description: Allow read, run and clear rootcheck information
+    policies:
+      syscheck:
+        actions:
+          - rootcheck:clear
+          - rootcheck:read
+          - syscheck:run
+        resources:
+          - agent:id:*
+        effect: allow
```

Comment:
We need to add these new entities to the `api/test/integration/env/configurations/security/manager/schema_security_test.sql`. And them execute security tests